Tests/fixtures/podcasting.yml

```diff
@@ -0,0 +1,9536 @@
+  browser_family: Unknown
+
+-
+  user_agent: 'Newsly'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Newsly
+    version: ""
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'NRC Audio/2.0.0 (nl.nrc.audio; build:29; Android 12; Sdk:31; Manufacturer:samsung; Model: SM-G975F) OkHttp/4.9.3'
+  os:
+    name: Android
+    version: "12"
+    platform: ""
+  client:
+    type: mobile app
+    name: NRC Audio
+    version: ""
+  device:
+    type: smartphone
+    brand: Samsung
+    model: 'Galaxy S10+'
+  os_family: Android
+  browser_family: Unknown
+
+-
+  user_agent: 'NRC%20Audio/1 CFNetwork/1410.0.3 Darwin/22.6.0'
+  os:
+    name: iOS
+    version: "16.6"
+    platform: ""
+  client:
+    type: mobile app
+    name: NRC Audio
+    version: "%20"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'iPad (compatible; Tablet2.0) NRC Digitale Editie, nl.nrc.nrcapp 10.0.7 (197) / iPadOS 16.6'
+  os:
+    name: iOS
+    version: ""
+    platform: ""
+  client:
+    type: mobile app
+    name: NRC
+    version: ""
+  device:
+    type: tablet
+    brand: Apple
+    model: 'iPad'
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'iPhone (compatible; Smartphone2.0) NRC Digitale Editie, nl.nrc.nrcapp 10.0.7 (197) / iOS 16.6'
+  os:
+    name: iOS
+    version: "16.6"
+    platform: ""
+  client:
+    type: mobile app
+    name: NRC
+    version: ""
+  device:
+    type: smartphone
+    brand: Apple
+    model: 'iPhone'
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'NRC-Nieuws/10.2.1 (nl.nrc.nrcapp; build:206; iOS 16.3.1) Alamofire/10.2.1'
+  os:
+    name: iOS
+    version: "16.3.1"
+    platform: ""
+  client:
+    type: mobile app
+    name: NRC
+    version: ""
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Outcast/1957 CFNetwork/889.7 Darwin/17.2.0'
+  os:
+    name: iOS
+    version: "11.1"
+    platform: ""
+  client:
+    type: mobile app
+    name: Outcast
+    version: "1957"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Outcast for Apple Watch'
+  os:
+    name: watchOS
+    version: ""
+    platform: ""
+  client:
+    type: mobile app
+    name: Outcast
+    version: ""
+  device:
+    type: wearable
+    brand: Apple
+    model: 'Watch'
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Outcast 3.1.9'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Outcast
+    version: "3.1.9"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'Podcast Overhaul/11.8 Android/9  https://www.podcastoverhaul.com/'
+  os:
+    name: Android
+    version: "9"
+    platform: ""
+  client:
+    type: mobile app
+    name: Overhaul
```

Comment:
From what I understand they are now https://www.overhaul.fm/ and their branding does include "FM". Perhaps this item and the ones below should be "Overhaul FM"?